source/wp-content/plugins/wporg-developer-blog-snippets/index.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/**
+ * Plugin Name: Developer Blog Snippets
+ * Plugin URI:  https://github.com/WordPress/wporg-developer-blog
+ * Description: Custom post type and taxonomy to introduce snippets to the developer blog.
+ * Version: 1.0.0
+ * Author: WordPress.org
+ * Author URI: https://wordpress.org/
+ * License:      GPL2
+ * License URI:  https://www.gnu.org/licenses/gpl-2.0.html
+ * Text Domain: wporg
+ */
+
+/**
+ * Register the CPT and tax for snippets.
+ */
+add_action(
+	'init',
+	function() : void {
+		$post_type = 'snippets';
+		$lang_tax  = 'coding-lang';
+		$api_tax   = 'api';
+
+		// Register the custom post type.
+		$cpt_args = [
+			'labels'       => [
+				'name'          => _x( 'Snippets', 'Post type general name', 'wporg' ),
+				'singular_name' => _x( 'Snippet', 'Post type singular name', 'wporg' ),
+				'add_new'       => __( 'Add New Snippet', 'wporg' ),
+			],
+			'public'       => true,
+			'has_archive'  => true,
+			'hierarchical' => false,
+			'show_in_rest' => true,
+			'menu_icon'    => 'dashicons-editor-code',
+			'supports'     => [ 'title', 'editor', 'author', 'thumbnail' ],
+			'template'     => [
+				[
+					'core/paragraph',
+					[
+						'placeholder' => __( 'Describe your use case for this snippet', 'wporg' )
```

Comment:
Minor code standards issue, array items should end with a comma.
```suggestion
						'placeholder' => __( 'Describe your use case for this snippet', 'wporg' ),
```